templates/compose/signoz.yaml

```
@@ -0,0 +1,627 @@
+              timeout: 2s
+            signozspanmetrics/delta:
+              metrics_exporter: signozclickhousemetrics
+              metrics_flush_interval: 60s
+              latency_histogram_buckets: [100us, 1ms, 2ms, 6ms, 10ms, 50ms, 100ms, 250ms, 500ms, 1000ms, 1400ms, 2000ms, 5s, 10s, 20s, 40s, 60s ]
+              dimensions_cache_size: 100000
+              aggregation_temporality: AGGREGATION_TEMPORALITY_DELTA
+              enable_exp_histogram: true
+              dimensions:
+                - name: service.namespace
+                  default: default
+                - name: deployment.environment
+                  default: default
+                # This is added to ensure the uniqueness of the timeseries
+                # Otherwise, identical timeseries produced by multiple replicas of
+                # collectors result in incorrect APM metrics
+                - name: signoz.collector.id
+                - name: service.version
+                - name: browser.platform
+                - name: browser.mobile
+                - name: k8s.cluster.name
+                - name: k8s.node.name
+                - name: k8s.namespace.name
+                - name: host.name
+                - name: host.type
+                - name: container.name
+          extensions:
+            health_check:
+              endpoint: 0.0.0.0:13133
+            pprof:
+              endpoint: 0.0.0.0:1777
+          exporters:
+            clickhousetraces:
+              datasource: tcp://clickhouse:9000/signoz_traces
+              low_cardinal_exception_grouping: ${env:LOW_CARDINAL_EXCEPTION_GROUPING}
+              use_new_schema: true
+            signozclickhousemetrics:
+              dsn: tcp://clickhouse:9000/signoz_metrics
+            clickhouselogsexporter:
+              dsn: tcp://clickhouse:9000/signoz_logs
+              timeout: 10s
+              use_new_schema: true
+          service:
+            telemetry:
+              logs:
+                encoding: json
+            extensions:
+              - health_check
+              - pprof
+            pipelines:
+              traces:
+                receivers: [otlp]
+                processors: [signozspanmetrics/delta, batch]
+                exporters: [clickhousetraces]
+              metrics:
+                receivers: [otlp]
+                processors: [batch]
+                exporters: [signozclickhousemetrics]
+              metrics/prometheus:
+                receivers: [prometheus]
+                processors: [batch]
+                exporters: [signozclickhousemetrics]
+              logs:
+                receivers: [otlp]
+                processors: [batch]
+                exporters: [clickhouselogsexporter]
+      - type: bind
+        source: ./otel-collector-opamp-config.yaml
+        target: /etc/manager-config.yaml
+        content: |
+          server_endpoint: ws://signoz:4320/v1/opamp
+    environment:
+      - SERVICE_FQDN_OTELCOLLECTORHTTP_4318
```

services now need to use the `SERVICE_URL` variable instead of `SERVICE_FQDN` since the latest update from a few days ago.
```suggestion
      - SERVICE_URL_OTELCOLLECTORHTTP_4318
```